hw/ip/adc_ctrl/adc_ctrl.core

```diff
@@ -2,8 +2,8 @@ CAPI=2:
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:ip:dcd:1.0"
-description: "dcd"
+name: "lowrisc:ip:adc_ctrl:1.0"
+description: "adc_ctrl"
```

Comment:
Can you maybe expand the description a bit to describe what adc ctrl is?

Reply:
ADC means analog to digital converter.
CTRL means controller.
This module is used for USB-C debug cable detection(CC1/CC2 voltage sensing).

Reply:
I'm sorry for the confusion. I meant changing the adc_ctrl.core file to put what you just mentioned in the `description` key.

For example:

```suggestion
description: "USB-C debug cable detection (CC1/CC2 voltage sensing)"
```

Reply:
Sure. I can do that.